client/services/FeedbackGeneratorServiceSpec.js

```diff
@@ -26,6 +26,27 @@ describe('FeedbackGeneratorService', function() {
     FeedbackGeneratorService = $injector.get('FeedbackGeneratorService');
   }));
 
+  describe('_jsToHumanReadable', function() {
+    it('should return "stringified" (readable) versions of input variables', 
+      function() {
+      expect(
+        FeedbackGeneratorService._jsToHumanReadable('cat')
+      ).toEqual('"cat"');
+      expect(
+        FeedbackGeneratorService._jsToHumanReadable(1)
+      ).toEqual('1');
+      expect(
+        FeedbackGeneratorService._jsToHumanReadable(true)
```

Comment:
Darn, I just realized that the expected value here is language dependent. Can you add a TODO for me (at jsToHumanReadable) to change the human-readable representation based on language? (E.g. true wouldn't be capitalized if the user was programming in JS.)